### DISCUSSION TITLE
Why did the Robo_Advisor disappear?

### DISCUSSION THREAD

**Discussion by zj-nb** [AI4Finance-Foundation/FinGPT#101] [Q&A]
I looked at it yesterday and found that Robo_Advisor was deleted, why is that? Can anyone know the reason？Please

**Comment by BruceYanghy** [comment]
https://github.com/AI4Finance-Foundation/FinGPT/tree/master/fingpt/FinGPT_v2/FinGPT_Robo_Advisor

**Comment by BruceYanghy** [comment]
It's under development, we are about to train a chat version LLMs specifically for Robo-advisor
